_posts/0000-01-02-HugAndHerBen.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
This is a quote from Plato's Republic "In the end, all end when it has to end because the end of all is not clear to those who do not see the end of all"